go.mod

```diff
@@ -24,30 +24,50 @@ require (
 	github.com/Azure/go-autorest/autorest/date v0.3.0 // indirect
 	github.com/Azure/go-autorest/logger v0.2.1 // indirect
 	github.com/Azure/go-autorest/tracing v0.6.0 // indirect
+	github.com/PuerkitoBio/purell v1.1.1 // indirect
```

Comment:
@alculquicondor @ArangoGutierrez any idea where those changes came from? I had to run `o get sigs.k8s.io/kustomize/kustomize/v4` to deploy, so probably from that; is it expected?

Reply:
you shouldn't use `go get`, but `go install` instead. The new makefile does that.

Reply:
It is not working for me, make deploy still fails with 

```
no required module provides package sigs.k8s.io/kustomize/kustomize/v4; to add it:
        go get sigs.k8s.io/kustomize/kustomize/v4
make: *** [Makefile:183: kustomize] Error 1
```

Reply:
did you figure it out or did you just revert the file?

Reply:
Just reverted the file